db_scripts/mosip_esignet/deploy.sh

```
@@ -1,65 +1,54 @@
 echo `date "+%m/%d/%Y %H:%M:%S"` ": $properties_file"
-
 if [ -f "$properties_file" ]
 then
-	echo `date "+%m/%d/%Y %H:%M:%S"` ": Property file \"$properties_file\" found."
-	while IFS='=' read -r key value
-	do
-		key=$(echo $key | tr '.' '_')
-		eval ${key}=\${value}
-	done < "$properties_file"
+     echo `date "+%m/%d/%Y %H:%M:%S"` ": Property file \"$properties_file\" found."
+    while IFS='=' read -r key value
```

Shellcheck (0.11.0)

[warning] value appears unused. Verify use (or export if used externally).

(SC2034)